diff -aurN gnome-jabber-0.1.0.orig/Makefile.in gnome-jabber-0.1.0/Makefile.in
--- gnome-jabber-0.1.0.orig/Makefile.in	2003-05-14 19:23:37.000000000 +0200
+++ gnome-jabber-0.1.0/Makefile.in	2003-07-05 00:27:41.000000000 +0200
@@ -30,7 +30,7 @@
 	@cd src/ && $(MAKE) clean	
 
 install: src/gnome-jabber src/gnome-jabber.glade
-	$(INSTALL_DIR) $(pkgdatadir)
+	$(INSTALL_DIR) $(DESTDIR)$(pkgdatadir)
 	@cd src/ && $(MAKE) install
 	@cd images/ && $(MAKE) install
 
diff -aurN gnome-jabber-0.1.0.orig/images/Makefile.in gnome-jabber-0.1.0/images/Makefile.in
--- gnome-jabber-0.1.0.orig/images/Makefile.in	2003-05-14 19:23:39.000000000 +0200
+++ gnome-jabber-0.1.0/images/Makefile.in	2003-07-05 00:28:13.000000000 +0200
@@ -64,7 +64,7 @@
 
 install: $(OBJS)
 	for file in $(OBJS) ; \
-		do $(INSTALL_DATA) $$file $(pkgdatadir) ; \
+		do $(INSTALL_DATA) $$file $(DESTDIR)$(pkgdatadir) ; \
 	done
 
 uninstall: $(OBJS)
diff -aurN gnome-jabber-0.1.0.orig/src/Makefile.in gnome-jabber-0.1.0/src/Makefile.in
--- gnome-jabber-0.1.0.orig/src/Makefile.in	2003-07-02 02:06:31.000000000 +0200
+++ gnome-jabber-0.1.0/src/Makefile.in	2003-07-05 00:29:04.000000000 +0200
@@ -125,11 +125,11 @@
 	rm -f config.h Makefile *~ *.bak core
 
 install: all $(INSTALL_OBJS)
-	$(INSTALL_DIR) $(bindir)
-	$(INSTALL_PROGRAM) ./gnome-jabber $(bindir)/gnome-jabber
+	$(INSTALL_DIR) $(DESTDIR)$(bindir)
+	$(INSTALL_PROGRAM) ./gnome-jabber $(DESTDIR)$(bindir)/gnome-jabber
 
 	for file in $(INSTALL_OBJS) ; \
-                do $(INSTALL_DATA) $$file $(pkgdatadir) ; \
+                do $(INSTALL_DATA) $$file $(DESTDIR)$(pkgdatadir) ; \
         done
 
 
